packages/by-name/kata/runtime-class-files/package.nix

```diff
@@ -32,18 +54,19 @@ stdenvNoCC.mkDerivation {
   # TODO(msanft): perform the actual launch digest calculation.
   buildPhase = ''
     mkdir -p $out
-    sha256sum ${image} ${kernel} ${qemu-bin} ${qemu-share}/kvmvapic.bin ${qemu-share}/linuxboot_dma.bin ${qemu-share}/efi-virtio.rom ${containerd-shim-contrast-cc-v2} ${ovmf} | sha256sum | cut -d " " -f 1 > $out/launch-digest.hex
+    sha256sum ${image} ${kernel} ${qemu-snp.bin} ${qemu-tdx.bin} ${containerd-shim-contrast-cc-v2} ${ovmf-snp} ${ovmf-tdx} | sha256sum | cut -d " " -f 1 > $out/launch-digest.hex
```

Comment:
This removes the files in `share/` from the (admittedly fake) launch digest.

Reply:
I'm aware of this, but it can only ever actually get relevant if we have a change in the QEMU share directories, but not the QEMU binary itself, which I don't expect to happen before we get an actual launch digest. I think it'd be excessive to measure all of the components of both here.